Applications/Games/MagicTheGatheringArena/Online/script.js

```diff
@@ -0,0 +1,30 @@
+include("engines.wine.quick_script.custom_installer_script");
+include("engines.wine.verbs.dxvk");
+include("engines.wine.verbs.dotnet472"); // ALWAYS USE LATEST!
```

Comment:
Well, dotnet472 is not even working with winetricks, so I am not really sure of that one

Reply:
> dotnet472 is not even working with winetricks, so I am not really sure of that one

Doubt, works on my machine

Reply:
I can confirm dotnet472 installation works.

Reply:
It executes, but if you look at what wine is doing, the execution of the dotnet installation files stops after some lines.

Reply:
@ImperatorS79 well make a new issue for that on either winetricks or phoenicis then.. -> resolved for the time beeing

Reply:
@ImperatorS79 + we don't even need dotnet472 afaik it was suggested on WineHQ, but i excluded it in https://github.com/PhoenicisOrg/scripts/issues/934

Reply:
@ImperatorS79 I took a quick look at POL5 script and winetricks script for dotnet472 and I think it's because winetricks uses an un-official repack of the .exe file

Reply:
Still not relevant to this merge request.. make issue for it if dotnet472 has issues in phoenicis..

~~stop spamming my MR! :(~~